core/shared/src/main/scala/cats/effect/unsafe/IORuntime.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cats.effect.unsafe
+
+import scala.concurrent.ExecutionContext
+
+@annotation.implicitNotFound("""Could not find an implicit IORuntime.
+
+Instead of calling unsafe methods directly, consider using cats.effect.IOApp, which
+runs your IO. If integrating with non-functional code or experimenting in a REPL / Worksheet,
+add the following import:
+
+import cats.effect.unsafe.implicits.global
+
+Alternatively, you can create an explicit IORuntime value and put it in implicit scope.
```

Comment:
I would add here something like: "This may be useful if you have a pre-existing fixed thread pool and/or scheduler which you wish to use to execute IO. Please be sure to review thread pool best practices to avoid unintentionally degrading your application performance."